charts/seed-terraformer/charts/alicloud-dns/values.yaml

```diff
@@ -3,6 +3,7 @@ record:
   hostRecord: my-dns-record
   value: 1.2.3.4
   type: ip
+  ttl: "600"
```

Comment:
OK - just out of curiosity - why do you need `600`?

Reply:
This is a long story.
1. I use my personal account to test dns. Ttl for personal account is **no** less than 600s. 
2. For commany account, there is long process to apply a Domain.. Maybe the least ttl could be 120s. Not verified. 
3. I could log an issue. Current value is just for happy pass.

Reply:
OK, thx.